test/common/testlib.py

```diff
@@ -231,6 +231,11 @@ def key_press(self, keys):
             else:
                 self.cdp.invoke("Input.dispatchKeyEvent", type="char", text=k)
 
+    def set_input_text(self, selector, val):
```

Comment:
I would explicitly mention "type" in the name of this function

Reply:
I'd say "typing" is an implementation detail.  If we later find a better way that doesn't involve key-press events, I think we should use it.